src/app/backend/dashboard.go

```diff
@@ -61,6 +61,7 @@ func main() {
 	// TODO(bryk): Disable directory listing.
 	http.Handle("/", http.FileServer(http.Dir("./public")))
 	http.Handle("/api/", CreateHttpApiHandler(apiserverClient, heapsterRESTClient, config))
+	http.Handle("/api/appConfig.js", AppHandler(configHandler))
```

Comment:
Temporarily using `/api/*` path to avoid problems when running on `serve` instead of `serve:prod`. I'll try to use tip from #640.